cmd/ftl/cmd_serve.go

```diff
@@ -55,7 +55,10 @@ func (s *serveCmd) Run(ctx context.Context) error {
 			_ = KillBackgroundServe(logger)
 		}
 
-		runInBackground(logger)
+		if err := runInBackground(logger); err != nil {
+			logger.Errorf(err, "runInBackground failed")
```

Comment:
You might be able to remove the log message since the cmd will error as well. Otherwise, I think we might get duplicate error messages like this:
![Screenshot 2024-04-25 at 10 57 11 AM](https://github.com/TBD54566975/ftl/assets/51647/18cde998-9f76-4ad0-95c6-ebe89507b5ba)

Reply:
removed